src/config/mod.rs

```diff
@@ -279,6 +285,12 @@ impl Resource {
     }
 }
 
+impl From<SocketAddr> for Resource {
+    fn from(addr: SocketAddr) -> Self {
+        Self::Port(addr.port())
```

Comment:
I'm not sure if this really matters or not, but couldn't a system with two different network interfaces have listeners on the same port number on each without conflicting? To handle that I think we'd need to do some translation of `0.0.0.0` as "all interfaces" and probably some other things as well. Likely not worthwhile.

Reply:
:+1: it does seem like we should be considering the address with the port.

I think the only special consideration will be `0.0.0.0` and `::` for all addresses.

Reply:
True, good catch. It should be just a matter of implementing custom comparison traits.